client/src/components/Authentication/AuthLandingPage.js

```diff
@@ -1,7 +1,7 @@
-
+import Login from '../Login/Login';
 
 export default function AuthLandingPage() {
     return (
-        <h1>Test</h1>
+        <Login />
```

Comment:
Works fine but consider future development.  Since we will likely have other components on this page, we would want this wrapped in a div/container/etc.  No change needed at the moment.